src/handle/command.ml

```diff
@@ -351,18 +353,20 @@ let handle : (Path.t -> Sign.t) -> sig_state -> p_command ->
      | _, _, _, _ -> ());
     (* Build proof data. *)
     let data =
-      (* Scope keeping position, with [expo] parsed above. *)
-      let scope_p a = Pos.make a.pos (scope expo a) in
+      (* Scope keeping position, with [expo] parsed above without warning . *)
+      let scope_p warn a = Pos.make a.pos (scope ~warn expo a) in
```

Comment:
Why not writing:
```
let warn = p_sym_arg <> [] && p_sym_typ <> None && p_sym_trm <> None in
let scope_p a = Pos.make a.pos (scope ~warn expo a)
```

Reply:
We want to remove warning when the list of arguments is not empty, and we have both a type and a definition, which would render `warn = not (p_sym_arg <> [] && p_sym_typ = Some _ && p_sym_trm = Some _)`, so `(p_sym_arg <> [] || p_sym_trm = None || p_sym_typ = None)`? Or am I missing something?

Reply:
p_sym_arg = []

Reply:
It's even worse, I think we want `p_sym_arg = [] || p_sym_typ = None && p_sym_trm <> None || p_sym_typ <> None && p_sym_trm = None`, namely, we want to use normal warning if there is only a type or only a term, and if there is no argument, then we use normal warning anyway.

Reply:
I don't understand. We want warnings when there is no parameters or, if there are parameters, when there is no type or no definition. No need to check that, when there is no type, there is a definition, and vice versa, because there must be one.

Reply:
Ah indeed.